README.md

```diff
@@ -59,6 +74,8 @@ sizes:
 * **images** is a list of image names (as used by `singleuser_image_spec` option in KubeSpawner)
 * **users** is a list of users allowed to use this profile, to ignore user filtering, specify `"*"` as a user name, or completely omit the `users` section
 * **env** is an object containing environment variables to be set for a singleuser server. *Keep in mind that all the values need to be strings - i.e. have quotes numbers!*
+* **node_tolerations** is a list of OpenShift node tolerations to be applied to the singleuser pod. See https://docs.openshift.com/container-platform/3.11/admin_guide/scheduling/taints_tolerations.html#admin-guide-taints for more information.
```

Comment:
I would prefer linking either to the latest OpenShift docs (4.3) or maybe even to Kubernetes documentation for this

Reply:
Fixed.